docs/HowToLogInLocally.md

```diff
@@ -0,0 +1,27 @@
+# Authentication in your Development Env. 
+
+In order to be able to log in to your local environment, you will need to do the following: 
+
+## Prerequisites
+
+- Have the [site](https://github.com/department-of-veterans-affairs/vets-website) building locally, and the drupal content updated. 
+- Have the [API](https://github.com/department-of-veterans-affairs/vets-api) running locally, with [mock-api-data](https://github.com/department-of-veterans-affairs/vets-api-mockdata), either in docker or locally
+
+
+## To log in:
+0) Have both the website and API running. 
+1) Head to [flipper for the API](http://localhost:3000/flipper/features)
+2) Disable the `ssoe` feature flag. 
```

Comment:
Can we remove these lines?
SSOe should be working locally now, and it'll be the only option in the near future (e.g. the `ssoe` feature flags will be removed and everyone will use SSOe login).
If you're having trouble with SSOe login, let me know!

Reply:
```suggestion
```